tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -193,6 +194,22 @@ Verify JupyterHub Pod Logs Dont Have Errors About Distutil Library
     ...       ODS-586
     Verify Errors In Jupyterhub Logs
 
+Verify Grafana Is Connected To Prometheus Using TLS
+    [Documentation]    Verifies Grafana is connected to Prometheus using TLS
+    [Tags]    Tier2
+    ...       ODS-963
+    [Setup]  Set Library Search Order  Selenium Library
+    ${res} =  Oc Get  kind=Secret  name=grafana-datasources  namespace=redhat-ods-monitoring
```

Comment:
This line could be inside `Check TLS Is Enabled`:
`${res} =  Oc Get  kind=Secret  name=grafana-datasources  namespace=redhat-ods-monitoring`

Reply:
updated